### PR TITLE
Add secrets for auth utils deploy to maven.

### PR DESCRIPTION
**Pull request by MancunianSam**
Some of these will conflict with the other PRs but I can sort that out
later

### CODE REVIEW

File: root_data.tf [hunk 1]
```diff
@@ -29,3 +29,19 @@ data "aws_ssm_parameter" "workflow_pat" {
 data "aws_ssm_parameter" "slack_webhook_url" {
   name = "/mgmt/release/slack/webhook"
 }
+
+data "aws_ssm_parameter" "gpg_passphrase" {
+  name = "	/mgmt/github/gpg/passphrase"
```
Comment (TomJKing) [inline root_data.tf:34]: Is there a reason for the whitespace at the beginning of the names of the params?

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment.

File: root_data.tf (hunk 1 above)
Comment (MancunianSam) [reply root_data.tf:34]: I've removed this too.

Comment (TomJKing) [inline root_data.tf:34]: Still seeing some white space here for this param: `gpg_passphrase`

**Review by TomJKing — COMMENTED** [review]
One minor comment

**Review by TomJKing — APPROVED** [review]

**Pull request merged**